ea-defs/config.go

```diff
@@ -157,3 +158,32 @@ func RequiredEADefsFor(cloud_type string) (res []ibclient.EADefinition) {
 	}
 	return
 }
+
+//Checks for cloud license in nios
+func CheckForCloudLicense(objMgr *ibclient.ObjectManager) {
+	flag, err := CheckLicense(objMgr, "cloud")
+	if err != nil {
+		fmt.Println("error", err)
```

Comment:
In other place of this file uses 'logrus' use that

Reply:
done